### ISSUE TITLE
Form 21P-601: Accessibility issues -Conditionally revealed field doesn’t follow VADS requirements

### ISSUE THREAD

**Issue by nkutub** [department-of-veterans-affairs/va.gov-team#127781]
## BACKGROUND
Team BIO-HEART completed Staging Review for Form 21P-601 on 10/30/25. There were 38 total findings identified by VA Platform (8 were launch blocking, while 30 were non-launch blocking). 

## PROBLEM STATEMENT / GOAL

- Page: Beneficiary’s other debts
- Device: Mobile (or desktop with a small browser window)
- Orientation: Portrait
On mobile (or mobile-sized) devices, the “Please describe the other debts” conditional field doesn’t follow VADS design practices. It’s not clear that the field is associated with answering “Yes”, visually. And the trigger option should be placed last so that the conditional field is the next one the user encounters: [https://design.va.gov/patterns/ask-users-for/relationship#conditionally-revealed-fields](https://design.va.gov/patterns/ask-users-for/relationship%23conditionally-revealed-fields) 
Beneficiary’s other debts:

<img width="602" height="834" alt="Image" src="https://github.com/user-attachments/assets/2b6c47f0-6b16-49c3-9afb-8387571be1a8" />

<img width="936" height="312" alt="Image" src="https://github.com/user-attachments/assets/1ebf415d-6ee3-4c9f-82ea-170b387202c4" />

**Issue Severity**
- Expected WCAG failure: n/a (not a WCAG issue)
- Expected 508 audit severity: n/a
- Expected CC severity level: a11y-defect-3 (These issues have the potential to disrupt the user experience)
- Expected experience standard: User encounters consistent, clear design components and patterns

**How to recreate:**
- Answer “yes” on the “Beneficiary’s other debts” page

**Recommendation:** 
Follow the [VADS guidelines](https://staging.va.gov/mock-form-patterns/relationship-to-veteran) for conditional fields - make the trigger the last option, and style it in the same way (blue border + indent).
